manywheel/build_common.sh

```diff
@@ -85,119 +130,141 @@ popd
 # so manually do the work of copying dependency libs and patchelfing
 # and fixing RECORDS entries correctly
 ######################################################################
-yum install -y zip openssl
 
 fname_with_sha256() {
     HASH=$(sha256sum $1 | cut -c1-8)
     DIRNAME=$(dirname $1)
     BASENAME=$(basename $1)
     if [[ $BASENAME == "libnvrtc-builtins.so" ]]; then
-	      echo $1
+              echo $1
     else
-	      INITNAME=$(echo $BASENAME | cut -f1 -d".")
-	      ENDNAME=$(echo $BASENAME | cut -f 2- -d".")
-	      echo "$DIRNAME/$INITNAME-$HASH.$ENDNAME"
+              INITNAME=$(echo $BASENAME | cut -f1 -d".")
+              ENDNAME=$(echo $BASENAME | cut -f 2- -d".")
+              echo "$DIRNAME/$INITNAME-$HASH.$ENDNAME"
     fi
 }
 
 make_wheel_record() {
     FPATH=$1
     if echo $FPATH | grep RECORD >/dev/null 2>&1; then
-	# if the RECORD file, then
-	echo "$FPATH,,"
+        # if the RECORD file, then
+        echo "$FPATH,,"
     else
-	HASH=$(openssl dgst -sha256 -binary $FPATH | openssl base64 | sed -e 's/+/-/g' | sed -e 's/\//_/g' | sed -e 's/=//g')
-	FSIZE=$(ls -nl $FPATH | awk '{print $5}')
-	echo "$FPATH,sha256=$HASH,$FSIZE"
+        HASH=$(openssl dgst -sha256 -binary $FPATH | openssl base64 | sed -e 's/+/-/g' | sed -e 's/\//_/g' | sed -e 's/=//g')
+        FSIZE=$(ls -nl $FPATH | awk '{print $5}')
+        echo "$FPATH,sha256=$HASH,$FSIZE"
     fi
 }
 
 mkdir -p /$WHEELHOUSE_DIR
 cp $PYTORCH_DIR/$WHEELHOUSE_DIR/*.whl /$WHEELHOUSE_DIR
+if [[ -n "$BUILD_PYTHONLESS" ]]; then
+    mkdir -p /$LIBTORCH_HOUSE_DIR
+    cp $PYTORCH_DIR/$LIBTORCH_HOUSE_DIR/*.zip /$LIBTORCH_HOUSE_DIR
+fi
 mkdir /tmp_dir
 pushd /tmp_dir
 
-for whl in /$WHEELHOUSE_DIR/torch*linux*.whl; do
+for pkg in /$WHEELHOUSE_DIR/torch*linux*.whl /$LIBTORCH_HOUSE_DIR/libtorch*.zip; do
+
+    # if the glob didn't match anything
```

Comment:
fixes the bug from last time, as verified by https://ci.pytorch.org/jenkins/job/pip-packages/job/pip-cp27-cp27m-linux-build-upload/55/console